open-metadata-distribution/open-metadata-assemblies/pom.xml

```diff
@@ -0,0 +1,92 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!-- SPDX-License-Identifier: Apache-2.0 -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+    <parent>
+        <artifactId>open-metadata-distribution</artifactId>
+        <groupId>org.odpi.egeria</groupId>
+        <version>0.1-SNAPSHOT</version>
+    </parent>
+
+    <name>Open Metadata Assemblies</name>
+    <description>
+        Packaged executables from Egeria.
+    </description>
+
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>open-metadata-assemblies</artifactId>
+
+    <profiles>
+        <profile>
+            <id>Windows</id>
```

Comment:
For now I'm not sure we have any platform specifics. If/when we get to the point of adding .cmd / powershell scripts for windows and shell scripts for linux/mac, then this is a mechanism to get the right things included -- but I don't think we have any now, nor do we have python scripts, so I'd be inclined to remove them to avoid clutter - we can add once we identify a need?